### PR TITLE
fix(ui):  broken breadcrumb issue in request/update pages for tags/description

### LINKED ISSUE #13462: Broken breadcrumb in request/update pages for tag/description
**Affected module**
UI

**Describe the bug**
- Multiple slash can be seen in  breadcrumb in request/update pages for tag/description

**To Reproduce**

Screenshots or steps to reproduce

image

**Expected behavior**
- Only breadcrumb should be visible 

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes #13462

- broken breadcrumb issue in request/update pages for tags/description
- remove unwanted localization keys

image

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx [hunk 1]
```diff
@@ -165,6 +167,10 @@ const RequestDescription = () => {
     });
   }, [entityData]);
 
+  if (isEmpty(entityData)) {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx:170]: Good to use loading flag

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestTagPage/RequestTagPage.tsx [hunk 2]
```diff
@@ -155,6 +157,10 @@ const RequestTag = () => {
     });
   }, [entityData]);
 
+  if (isEmpty(entityData)) {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestTagPage/RequestTagPage.tsx:160]: same

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateDescriptionPage/UpdateDescriptionPage.tsx [hunk 3]
```diff
@@ -181,6 +182,10 @@ const UpdateDescription = () => {
     setCurrentDescription(getDescription());
   }, [entityData, columnObject]);
 
+  if (isEmpty(entityData)) {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateDescriptionPage/UpdateDescriptionPage.tsx:185]: same

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateTagPage/UpdateTagPage.tsx [hunk 4]
```diff
@@ -194,6 +195,10 @@ const UpdateTag = () => {
     setSuggestion(getTags());
   }, [entityData, columnObject]);
 
+  if (isEmpty(entityData)) {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateTagPage/UpdateTagPage.tsx:198]: same

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx:170]: @chirag-madlani  we are fetching the data from the utils function and setting the state there . image

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestTagPage/RequestTagPage.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestTagPage/RequestTagPage.tsx:160]: https://github.com/open-metadata/OpenMetadata/pull/13463#discussion_r1348736991

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateDescriptionPage/UpdateDescriptionPage.tsx (hunk 3 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateDescriptionPage/UpdateDescriptionPage.tsx:185]: https://github.com/open-metadata/OpenMetadata/pull/13463#discussion_r1348736991

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateTagPage/UpdateTagPage.tsx (hunk 4 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateTagPage/UpdateTagPage.tsx:198]: https://github.com/open-metadata/OpenMetadata/pull/13463#discussion_r1348736991

**Pull request merged**